src/k8s/pkg/k8sd/setup/kube_proxy_test.go

```diff
@@ -109,4 +109,30 @@ func TestKubeProxy(t *testing.T) {
 		g.Expect(err).To(BeNil())
 		g.Expect(val).To(BeEmpty())
 	})
+
+	t.Run("IPv6", func(t *testing.T) {
+		g := NewWithT(t)
+
+		// Create a mock snap
+		s := mustSetupSnapAndDirectories(t, setKubeletMock)
+		s.Mock.Hostname = "dev"
+
+		g.Expect(setup.KubeProxy(context.Background(), s, "dev", "fd98::/108", "[::1]", nil)).To(BeNil())
```

Comment:
> drive-by fix: brackets around the IPv6 are only required/accepted if combined with a port.

Isn't `[::1]` in conflict with the above statement?

Reply:
For the first comment: This is not our convention but the actual IP standard.

For the second: Well, inside the KubeProxy function we assemble this with the corresponding secure port. I agree that this is not nice as it is a somewhat implicit assumption of the implementation. We should probably revisit this in the future.